SQLDataSource.test.js

```diff
@@ -66,6 +39,7 @@ describe("PostgreSQL", () => {
       constructor() {
         super();
         this.db = knex;
+        this.hashing = true;
```

Comment:
I may be a bit confused, but if the constructor is setting this.hashing based on the config object... why is this necessary at all?

Reply:
if you want use in-memory cache you don`t need hashing